vector/src/main/res/menu/menu_timeline.xml

```diff
@@ -3,6 +3,11 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <item
+        android:id="@+id/search"
+        android:title="@string/search"
+        app:showAsAction="ifRoom" />
```

Comment:
We need an icon if it is shown as action then

Reply:
Done.